orchestrator-salt/src/main/resources/salt-common/salt/fluent/init.sls

```diff
@@ -4,7 +4,7 @@
 {% set dbus_lock_exists = salt['file.file_exists' ]('/etc/td-agent/databus_bundle.lock') %}
 
 {% if fluent.enabled %}
-
+{% if not fluent.fluent_prewarm_role in grains.get('roles', []) %}
```

Comment:
i do not think that is needed as if the /etc/td-agent folder exists, it's already sure the td-agent service is installed.

Reply:
Don't think that is the case. The dir check also wraps the plugin installation, which runs without this change.

Reply:
but because of that is wrapped inside the file check, it only runs if the dir is not there (so only runs for base images)

Reply:
That is not my observation.

Reply:
@oleewere so what do you think about this?